static/app/views/settings/account/notifications/notificationSettingsByTypeV2.tsx

```diff
@@ -332,14 +332,16 @@ class NotificationSettingsByTypeV2 extends DeprecatedAsyncComponent<Props, State
             fields={this.getFields()}
           />
         </Form>
-        <Form
-          saveOnBlur
-          apiMethod="PUT"
-          apiEndpoint="/users/me/notification-providers/"
-          initialData={this.getProviderInitialData()}
-        >
-          <BottomJsonForm fields={this.getProviderFields()} />
-        </Form>
+        {notificationType !== 'reports' ? (
+          <Form
+            saveOnBlur
+            apiMethod="PUT"
+            apiEndpoint="/users/me/notification-providers/"
+            initialData={this.getProviderInitialData()}
+          >
+            <BottomJsonForm fields={this.getProviderFields()} />
+          </Form>
+        ) : null}
```

Comment:
Does this handle saving the data correctly? As in, will this set NotificationSettingProvider for email?

Reply:
@snigdhas Yes, I verified. It all goes to the control silo so it's fine. I think the problem is that to get the projects you have to pick the region silo but not an issue for the organization which has replication in the control silo.